src/search-schema/data/definitions/enums/assay_types.yaml

```diff
@@ -64,23 +64,41 @@ image_pyramid:
     contains-pii: false
     vitessce-hints: ['is_image', 'pyramid']
     
-IMC:
-    description: Imaging Mass Cytometry
-    alt-names: []
+IMC2D:
+    description: 2D Imaging Mass Cytometry
+    alt-names: ['IMC', '2D-IMC']
     primary: true
     contains-pii: false
     vitessce-hints: []
 
-3D-IMC:
+IMC3D:
     description: 3D Imaging Mass Cytometry
-    alt-names: []
+    alt-names: ['3D-IMC']
     primary: true
     contains-pii: false
     vitessce-hints: []
 
-IMC_pyramid:
-    description: Imaging Mass Cytometry [Image Pyramid]
-    alt-names: []
+IMC2D_pyramid:
+    description: 2D Imaging Mass Cytometry [Image Pyramid]
+    alt-names:
+      - 'IMC_pyramid'
+      - ['IMC', 'Image Pyramid']
+      - ['Image Pyramid', 'IMC']
+      - ['IMC2D', 'Image Pyramid']
+      - ['Image Pyramid', 'IMC2D']
+    primary: false
+    vis-only: true
+    contains-pii: false
+    vitessce-hints: ['pyramid']
+
+IMC3D_pyramid:
+    description: 3D Imaging Mass Cytometry [Image Pyramid]
+    alt-names:
```

Comment:
Where are these alternatives being used? If they show up in user facing parts of the portal, they should also be changed to use "3D" as the postfix rather than prefix wherever possible.

Reply:
Right- they should never show up in user-facing places.  The alt-names are all deprecated, but examples of metadata files using those names are still in the system.  The alt-name entry allows the deprecated name to be mapped to the correct name before the user sees it.

Reply:
The two element alt-names like ['IMC', 'Image Pyramid'] are meant to match database entries which list those two types, mapping them to the correct description "3D Imaging Mass Cytometry [Image Pyramid]".  The fact that we have image pyramid datasets with the single assay type 'Image Pyramid' and the pair of assay types is an artifact of the fact that 'vis lifting' came along late in the game.